backstory/src/test/java/com/google/sps/servlets/AnalyzeImageServletTest.java

```diff
@@ -0,0 +1,148 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import static org.mockito.Mockito.*;
+import com.google.sps.images.ImagesManager;
+import com.google.sps.images.VisionImagesManager;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.sps.images.data.AnnotatedImage;
+import com.google.sps.perspective.PerspectiveStoryAnalysisManager;
+import com.google.sps.perspective.StoryAnalysisManager;
+import com.google.sps.perspective.data.StoryDecision;
+import com.google.sps.perspective.data.NoAppropriateStoryException;
+import com.google.sps.perspective.data.APINotAvailableException;
+import com.google.sps.story.PromptManager;
+import com.google.sps.story.StoryManager;
+import com.google.sps.story.StoryManagerImpl;
+import com.google.sps.servlets.AnalyzeImageServlet;
+import com.google.appengine.api.datastore.DatastoreService;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.ArgumentCaptor;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Map;
+import java.util.HashMap;
+import com.google.sps.perspective.data.APINotAvailableException;
+import com.google.sps.perspective.data.NoAppropriateStoryException;
+import com.google.sps.perspective.data.StoryDecision;
+
+/**
+ * Tests for the analyze image servlet, which contains all image analysis and backstory generation
+ * functionalities.
+ */
+// @RunWith(MockitoJUnitRunner.class)
+// public final class AnalyzeImageServletTest {
+
+  /**
+   * Tests that the correct input is passed from the initial call to the managers, 
+   * and that the correct ouput is passed from the managers to the permanent storage service.
+   */
+  // @Test
```

Comment:
I get that you've commented all this code out because you've changed implementations. You could also just add `@Ignore` next to `@Test` and it will skip that test. However, you don't need to change this because I know you'll be changing it later.